test/gen_rpc/multi_rpc_functional_SUITE.erl

```diff
@@ -0,0 +1,371 @@
+    Msg = Name = 'safeevalmfa1', 
+    ok = clean_process(Name, 'normal'),
+    TestPid = self(),
+    Pid = spawn_listener(?SLAVE1, Name, TestPid, 1),
+    true = register(Name, Pid),
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_one_node] Registered Listening Node"),
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, 'gen_rpc_test_helper', ping, [{?NODE, Name, Msg}]),
+    {ok, ?SLAVE1, passed} = wait_for_reply(?SLAVE1),
+    ok.
+
+safe_eval_everywhere_mfa_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2, ?FAKE_NODE],
+    Msg = Name = 'safeevalmfamulti', 
+    TestPid = self(),
+    ok = clean_process(Name, 'normal'),
+    Pid = spawn_listener2(?SLAVE1, ?SLAVE2, Name, TestPid, 2),
+    true = register(Name, Pid),
+    [true, [?FAKE_NODE]] = gen_rpc:safe_eval_everywhere(ConnectedNodes, 'gen_rpc_test_helper', ping, [{?NODE, Name, Msg}]),
+    {ok, passed} = wait_for_reply(?SLAVE1, ?SLAVE2),
+    ok.
+
+safe_eval_everywhere_mfa_multiple_nodes_TO(_Config) ->
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_multiple_nodes_TO]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    Msg = Name = 'safeevalmfamultiTO', 
+    TestPid = self(),
+    ok = clean_process(Name, 'normal'),
+    Pid = spawn_listener2(?SLAVE1, ?SLAVE2, Name, TestPid, 2),
+    true = register(Name, Pid),
+    SendTO = 10,
+    [true,[]] = gen_rpc:safe_eval_everywhere(ConnectedNodes, 'gen_rpc_test_helper', ping, [{?NODE, Name, Msg}], SendTO),
+    {ok, passed} = wait_for_reply(?SLAVE1, ?SLAVE2),
+    ok.
+
+safe_eval_everywhere_mfa_exit_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_exit_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, exit, ['fatal']),
+    % Nothing blows up on sender side after sending call to the ether
+    true = erlang:is_process_alive(whereis(gen_rpc_server_sup)),
+    true = erlang:is_process_alive(whereis(gen_rpc_acceptor_sup)),
+    true = erlang:is_process_alive(whereis(gen_rpc_client_sup)).
+
+safe_eval_everywhere_mfa_throw_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [safe_eval_everywhere_mfa_throw_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, throw, ['throwXup']),
+    ok = ct:pal("[erlang:throw only]. Verify the crash log from ct. You should see {{nocatch,throwXup}, ....} on the target node").
+
+safe_eval_everywhere_mfa_timeout_multiple_nodes(_Config) ->
+    ok = ct:pal("Testing [eval_everywhere_mfa_timeout_multiple_nodes]"),
+    ConnectedNodes = [?SLAVE1, ?SLAVE2],
+    [true, []] = gen_rpc:safe_eval_everywhere(ConnectedNodes, erlang, throw, ['throwXup']),
+    ok = ct:pal("erlang:throw only]. Verify the crash log from ct. You should see {{nocatch,throwXup}, ....} on the target node").
+
+client_inactivity_timeout(_Config) ->
```

Comment:
are u talking about the server and client inactivity tests?
If so they are dups and can be removed.